droid-api/src/test/java/uk/gov/nationalarchives/droid/internal/api/DroidAPITest.java

```diff
@@ -31,29 +31,38 @@
  */
 package uk.gov.nationalarchives.droid.internal.api;
 
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.rules.Timeout;
```

Comment:
This import is unused and the `java.util.Arrays` one underneath.

Reply:
I was trying to figure out the reason behind the test that runs a loop and at some point I thought, maybe that test was written for performance in mind hence thought of applying a timeout but decided against it, the import remained.

Reply:
Done